src/Bridges/Bridges.jl

```diff
@@ -41,7 +41,8 @@ MOIU.@model(AllBridgedConstraints,
 Returns a `LazyBridgeOptimizer` bridging `model` for every bridge defined in this package and for the coefficient type `T`.
 """
 function fullbridgeoptimizer(model::MOI.ModelLike, ::Type{T}) where T
-    bridgedmodel = MOIB.LazyBridgeOptimizer(model, AllBridgedConstraints{T}())
+    cache = MOIU.UniversalFallback(AllBridgedConstraints{T}())
```

Comment:
Are there any performance implications of adding another model layer?

Reply:
The UniversalFallback is designed to have almost no overhead for supported constraints.
By the way, this internal model will go away with light bridges so it does not have any long term implication.